code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -314,7 +314,7 @@
 
 				for(var/mob/living/L in view(7,src))
 
-					if(isslime(L) || L.stat == DEAD) // Ignore other slimes and dead mobs
+					if(L.stat == DEAD) // Ignore dead mobs
```

Comment:
out of curiosity, why did you remove this?

Reply:
Further down on line 327, the slime checks if the target mob has one of its own factions. Simple animal slimes get the "slime" faction which would mean they ignore other slimes. The check is redundant in this case.